packages/slate-react/src/components/android/android-input-manager.ts

```diff
@@ -0,0 +1,573 @@
+import { ReactEditor } from '../../plugin/react-editor'
+import { Editor, Node as SlateNode, Path, Range, Transforms } from 'slate'
+import { Diff, diffText } from './diff-text'
+import { DOMNode } from '../../utils/dom'
+import {
+  EDITOR_TO_ON_CHANGE,
+  EDITOR_TO_RESTORE_DOM,
+} from '../../utils/weak-maps'
+
+const debug = (...message: any[]) => {}
+
+/**
+ * Unicode String for a ZERO_WIDTH_SPACE
+ *
+ * @type {String}
+ */
+
+const ZERO_WIDTH_SPACE = String.fromCharCode(65279)
```

Comment:
I think the ZERO\_WIDTH\_SPACE hack is no longer used with the 0.50+ version of Slate and a search for "65279" doesn't appear to show any hits (at least in GitHub's search).

Maybe check to see if it's required and if not remove related code.

Reply:
Removed ZERO_WIDTH_SPACE and related code